libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -291,12 +291,17 @@ void VoteManager::verifyVotes(uint64_t pbft_round, size_t sortition_threshold, u
     }
     bool vote_is_valid = true;
     auto dpos_votes_count = dpos_eligible_vote_count(v->getVoterAddr());
-    if (v->getStep() == 1 && v->getType() == propose_vote_type) {
-      // We need to handle propose_vote_type
-      if (!dpos_votes_count) vote_is_valid = false;
-      dpos_votes_count = 1;
+    if (!dpos_votes_count) {
+      // After deep syncing, node could receive votes but still behind, may don't have vote sender state in table
+      LOG(log_nf_) << "Node may behind and still in syncing. Cannot find vote sender state in table " << v;
```

Comment:
I am not sure if I am ok with this - rather no than yes I guess. You handled 1 edge case when when node did not apply all synced pbft blocks yet - this will be happening very rarely at the start of node. But what about malicious/invalid votes ? I mean if you receive vote and voter has 0 dpos stake that should log error(or warning depends how we want to treat it). If you want to be so specific and log case when synced pbft blocks are not synced yet I guess you can check that explicitly ?

Reply:
"Node may behind and still in syncing." should be "Node may be still syncing".     But shouldn't the node KNOW if its still syncing?   And why is this an INFO level message?    In addition I think @JakubFornadel  makes valid points.